Error: `for` must be an `id` reference to a form control and may not reference other element types. To create the intended relationship, the `div` must have a `role` of "group" and an `aria-labelledby` attribute that references the label. Or use `fieldset` and `legend`. Error: Checkboxes in a group should use a `name` value that ends with "[]" to allow server-side code to collect the values as an array (or similar). Error: For broadest accessibility, implicitly associated `label` elements should also be explicitly associated with their control, using `for`.
diff --git a/output/Checkbox Group/4/8ea8d4a1-0dba-4c72-8525-bc2d2960b510.html b/output/Checkbox Group/4/8ea8d4a1-0dba-4c72-8525-bc2d2960b510.html
index aaed6bc..80a4335 100644
--- a/output/Checkbox Group/4/8ea8d4a1-0dba-4c72-8525-bc2d2960b510.html	
+++ b/output/Checkbox Group/4/8ea8d4a1-0dba-4c72-8525-bc2d2960b510.html	
@@ -1,12 +1,15 @@
-<label for="fruits">What fruits do you like?</label>
-<div id="fruits">
-  <label>
-    <input type="checkbox" name="fruits" value="bananas"> Bananas
+<label id="fruits">What fruits do you like?</label>
+<div role="group" aria-labelledby="fruits">
+  <label for="bananas">
+    <input type="checkbox" name="fruits[]" id="bananas" value="bananas">
+    Bananas
   </label>
-  <label>
-    <input type="checkbox" name="fruits" value="oranges"> Oranges
+  <label for="oranges">
+    <input type="checkbox" name="fruits[]" id="oranges" value="oranges">
+    Oranges
   </label>
-  <label>
-    <input type="checkbox" name="fruits" value="apples"> Apples
+  <label for="apples">
+    <input type="checkbox" name="fruits[]" id="apples" value="apples">
+    Apples
   </label>
 </div>
\ No newline at end of file
